pkg/skaffold/build/bazel/dependencies.go

```diff
@@ -83,3 +99,22 @@ func GetDependencies(ctx context.Context, workspace string, a *latest.BazelArtif
 func depToPath(dep string) string {
 	return strings.TrimPrefix(strings.Replace(strings.TrimPrefix(dep, "//"), ":", "/", 1), "/")
 }
+
+func findWorkspace(workingDir string) (string, error) {
+	dir, err := filepath.Abs(workingDir)
+	if err != nil {
+		return "", errors.Wrap(err, "invalid working dir")
+	}
+
+	for {
+		if _, err := os.Stat(filepath.Join(dir, "WORKSPACE")); err == nil {
+			return dir, nil
+		}
+
+		parent := filepath.Dir(dir)
+		if parent == dir {
+			return "", errors.New("no git repository found")
```

Comment:
should this message be something bazel-related instead of git-related?

Reply:
Lol. I’ll fix that tomorrow!